plugins/jira-status-rollup/README.md

```diff
@@ -0,0 +1,195 @@
+# Jira Status Rollup Plugin
```

Comment:
after addressing https://github.com/openshift-eng/ai-helpers/pull/6/files#r2430181607, I suggest with let this readme be the jira plugin readme, which sums up all the automation the plugin provides.
Then I don't need we need readme for each specific command, since the spec follow the hello-world conventions should be self explained enough.